NOTICE-3RD-PARTY-CONTENT.md

```diff
@@ -40,21 +40,17 @@
 ## Workflows
 | Dependency | Version | License |
 |:-----------|:-------:|--------:|
-|actions/checkout|v2|MIT License|
 |actions/checkout|v3|MIT License|
 |actions/upload-artifact|v2|MIT License|
 |actions/upload-artifact|v3.1.0|MIT License|
```

Comment:
Why you use both V2 and v3.1.0, I would suggest to use only v3.1.0

Reply:
Will be done in the next PR